pkgs/os-specific/linux/bionic-prebuilt/default.nix

```diff
@@ -0,0 +1,137 @@
+{ stdenvNoCC, lib, fetchurl, fetchzip, pkgs
+}:
+let
+
+  prebuilt_crt = fetchzip {
+    url = "https://android.googlesource.com/platform/prebuilts/gcc/linux-x86/aarch64/aarch64-linux-android-4.9/+archive/refs/heads/android11-release/lib/gcc/aarch64-linux-android/4.9.x.tar.gz";
+    sha256 = "1dshjv1557kf30nq4la4bph25ccpb713m9fvwi6w7wmvk0wgdc1c";
+    stripRoot = false;
+  };
+
+  prebuilt_libs = fetchzip {
+    url = "https://android.googlesource.com/platform/prebuilts/ndk/+archive/refs/heads/ndk-release-r22/platform/platforms/android-29/arch-arm64/usr/lib.tar.gz";
+    sha256 = "0pd8d11md55nd04vgpysgb47ajnnr4kabzdc08g15bvy90a7jsm8";
+    stripRoot = false;
+  };
+
+  prebuilt_ndk_crt = fetchzip {
+    url = "https://android.googlesource.com/platform/prebuilts/ndk/+archive/refs/heads/android11-release/r21/platforms/android-28/arch-arm64/usr/lib.tar.gz";
+    sha256 = "1g7s5gc69x42ryz8mmv4hzqzk985cs009d6s6kbkg550lcrfwvi7";
+    stripRoot = false;
+  };
+
+  ndk_support_headers = fetchzip {
+    url ="https://android.googlesource.com/platform/prebuilts/ndk/+archive/refs/heads/android11-release/r21/sources/cxx-stl/llvm-libc++/include/support.tar.gz";
+    sha256 = "1g7s5gc69x42ryz8mmv4hzqzk985cs009d6s6kbkg550lcrfwvi7";
+    stripRoot = false;
+  };
+
+  kernelHeaders = pkgs.makeLinuxHeaders {
+    arch = "arm64";
+    version = "5.1";
+    src = fetchurl {
+      url =
+        "https://android.googlesource.com/kernel/msm/+archive/db4bfbca77cf68e60e503f84074b853934bce55f.tar.gz";
+      sha256 = "0ki5gh1nsigjdpfpyw9ckbsrska3m94hdq1cxngmwif5rhv1z8vi";
+    };
+  };
+
+in
+stdenvNoCC.mkDerivation rec {
+  pname = "bionic-prebuilt";
+  version = "android11-release";
+
+  src = fetchurl {
+    url = "https://android.googlesource.com/platform/bionic/+archive/refs/heads/${version}.tar.gz";
+    sha256 = "0qklgmpy4x0bq34061d7grzsj0hgw8jbf937fda00a2ivz4l476c";
+  };
+
+  sourceRoot = ".";
+
+  NIX_DONT_SET_RPATH = true;
+
+  dontConfigure = true;
+  dontBuild = true;
+  patchPhase = ''
+    substituteInPlace libc/include/sys/cdefs.h --replace \
+      "__has_builtin(__builtin_umul_overflow)" "1"
+    substituteInPlace libc/include/bits/ioctl.h --replace \
+      "!defined(BIONIC_IOCTL_NO_SIGNEDNESS_OVERLOAD)" "0"
+  '';
+  installPhase= ''
+    # copy the bionic headers
+    echo copying libc files
```

Comment:
The comment and echo say different things.

Reply:
Done